### PR TITLE
data:Abhi03-tech

### PR DESCRIPTION
**Pull request by Abhi03-tech**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3361"></a>

### CODE REVIEW

**Comment by Abhi03-tech** [comment]
I guess, i have done all the steps correct this time. looking forward to it.

**Review by senali-d — CHANGES_REQUESTED** [review]
@Abhi03-tech Please rename the json file to github username.

`abhi3.json` ---> `data/Abhi03-tech.json`

Feel free to ask if you have any doubt.

**Comment by Abhi03-tech** [comment]
> @Abhi03-tech Please rename the json file to github username.
> 
> `abhi3.json` ---> `data/Abhi03-tech.json`
> 
> Feel free to ask if you have any doubt.

I have changed the file name. please check it out.

**Review by Panquesito7 — DISMISSED** [review]
Thanks! 🎉

File: data/Abhi03-tech.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/Abhi03-tech",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on HashNode",
+      "url": "https://hashnode.com/@Abhishek003",
+      "icon": "FaHashnode"
```
Comment (Aadarsh805) [inline data/Abhi03-tech.json:27]: Use `SiHashnode` for Hashnode icon, this one doesn't work

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/Abhi03-tech.json (hunk 1 above)
Comment (Abhi03-tech) [reply data/Abhi03-tech.json]: Done

**Review by Panquesito7 — APPROVED** [review]

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**